modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -87,29 +116,86 @@ data:
       - url: "https://remote-write.endpoint"
         basicAuth:
           username:
-            name: remoteWriteAuth
-            key: user
+            name: rw-basic-auth
+            key: <username>
           password:
-            name: remoteWriteAuth
-            key: password
+            name: rw-basic-auth
+            key: <password>
 ----
 +
-* The following example configures client TLS authentication:
+* *bearer token* 
++
+A bearer token (`bearerTokenFile`) configuration combines a basic authentication username and password with authorization credentials configured to use a specified token (such as an API key).
```

Comment:
I think the description is improper, since bearer token sets the `Authorization` header on every scrape request with the configured credentials.